finalPrep/modZeroHero.js

```diff
@@ -2,51 +2,89 @@
 
+var heroName = "Batman"
+var specialAbility = "Extreme Wealth and Privilege"
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = "Holy spamoni," +  heroName + "!"
+console.log(greeting);
+
+var catchphrase = `Alfred, the bathroom needs cleaning! I would do it but I have great responsibility due to my ${specialAbility}!`;
+console.log(catchphrase);
+
 // Declare two variables - power AND energy - set to integers
 
+var power = 4
+var energy = 50
+
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
+var fullPower =  power * 500
+
+var fullEnergy = energy +150
+
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
 
+var isHuman = true
+var identityConcealed = true
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
 
+var archEnemies = ["Riddleboy", "Cat Lady", "Jared Leto"]
+var sidekicks = ["Robin", "Batgirl", "Robin 2"]
+
 // Print the first sidekick to your console
 
+console.log(sidekicks[0]);
+
 // Print the last archEnemy to the console
+console.log(archEnemies[2]);
 
 // Write some code to add a new archEnemy to the archEnemies array
+archEnemies.unshift("The IRS");
 
 // Print the archEnemies array to console to ensure you added a new archEnemy
+console.log(archEnemies);
 
 // Remove the first sidekick from the sidekicks array
+sidekicks.shift()
 
 // Print the sidekicks array to console to ensure you added a new sidekick
+console.log(sidekicks);
 
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
 
 // Your function should include an if/else statement that meets the following criteria
 //   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
 //   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
 //   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
 
+function assessSituation(dangerLevel, saveTheDay, badExcuse) {
+  if (dangerLevel > 50) {
+    console.log(badExcuse);
+  } else if (dangerLevel > 10) {
```

Comment:
It is a best practice to not rely on order for conditions to avoid them overlapping (In this example, things greater than 50 are also greater than 10, so technically both conditions apply). You can think about having the most complicated logic in your `else` block.

Reply:
Would you restate the logic here? I kind of understand, but am having some confusion.

What I understand: That technically being greater than 50 and greater than 10 are not mutually exclusive, and hence something could fit both conditions.

What I'm confused by: "Having the most complicated logic in your else block"

I solved the problem I believe:

`function assessSituation(dangerLevel, saveTheDay, badExcuse) {
  if (dangerLevel > 50) {
    console.log(badExcuse);
  } else if (dangerLevel > 10 && dangerLevel <= 50) {
    console.log(saveTheDay);
  } else {
    console.log("Meh. Hard pass.");
  }
}`